spec/requests/swagger_docs/v6/full_assessment_spec.rb

```diff
@@ -413,17 +413,15 @@
                                    },
                                    tax: {
                                      type: :number,
-                                     maximum: 0,
                                      format: :decimal,
                                    },
-                                   benefits_in_kind: {
+                                   national_insurance: {
                                      type: :number,
-                                     minimum: 0,
```

Comment:
would be nice to replace this with maximum: 0 and a description explaining that it has to be negative